app/assets/stylesheets/responsive.scss

```diff
@@ -5,14 +5,15 @@
   .sidebar {
     width: 100%;
     hight: 200px;
```

Comment:
It seems you misspell the `height` attribute a little too frequently @andela-sadefila